client/src/index.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import { BrowserRouter as Router} from 'react-router-dom';
-
-
-ReactDOM.render(
-  <Router>
-    <App />
-  </Router>,
-  document.getElementById('root')
-);
-
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { Context } from "./components/DataManager"

```

Comment:
Needs to change to: import Provider from "./components/DataManager"

Alternatively, can be imported as : import DataManager from "./components/DataManager"